arnab_bhadra/arnabbhadra_PostgresAssignment/src/Entity/ClassSchedule.ts

```diff
@@ -0,0 +1,7 @@
+type ClassSchedule = {
```

Comment:
all this are type, better to keep it in service file, where we do most of the logic.

Reply:
When we use ORM like sequelize, we can create modals folder and keep our schema there. No need to do this now. Take a look at https://sequelize.org/master/